modules/compliance-profile-types.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-concepts/compliance-operator-understanding.adoc
+// * security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="compliance_profile_types_{context}"]
+= Compliance Operator profile types
+
+Compliance Operator rules are organized into profiles. Profiles can target the {product-title} Platform or Nodes, and some standards include `rhcos4` Node profiles.
+
+Platform:: Platform profiles evaluate your {product-title} cluster components. For example, confirming APIServer configurations are using strong encryption cyphers is Platform-level rule.
+
+Node:: Node profiles for evaluating the OpenShift or {op-system} configuration of each host. There are `ocp4` Node profiles and `rhcos4` Node profiles. `ocp4` Node profiles evaluate the OpenShift configuration of each host. For example, confirming kubeconfig files have the correct permissions to meet a compliance standard is a Node-level rule. `rhcos4` Node profiles evaluate the {op-system-first} configuration of each host. For example, confirming the SSHD service is configured to disable password logins is an {op-system}-level rule.

```

Comment:
At the very end of this sentence we're saying `RHCOS-level`, while referring to `Node-level` earlier in the sentence.

Do we want to replace `{op-system}-level` with `Node-level` so it's consistent? If we stick with `{op-system}-level`, it's technically a new term we haven't defined.